mule-extensions-api-persistence/src/test/java/org/mule/runtime/extension/api/persistence/BasePersistenceTestCase.java

```diff
@@ -127,7 +127,7 @@
   protected final ParameterDslConfiguration defaultParameterDsl = ParameterDslConfiguration.getDefaultInstance();
   protected final LayoutModel defaultLayoutModel = LayoutModel.builder().build();
   protected final ValueProviderModel defaultValueProviderModel =
-      new ValueProviderModel(emptyList(), false, false, false, 1, "ACategory", "AId");
+      new ValueProviderModel(emptyList(), emptyList(), false, false, false, 1, "ACategory", "AId");
```

Comment:
We shouldn't need to change this. This is because a constructor is being removed from the api.